tools/single_use/remove_publication_users_edit_perm_rel_duplicates.php

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env php
+<?php declare(strict_types=1);
+error_reporting(E_ALL);
```

Comment:
I don't see why this is here in this script. I don't think we override the error reporting level in any other script?